packages/riipen-ui/src/components/RadioButtonGroup.test.jsx

```diff
@@ -138,7 +138,7 @@ describe("<RadioButtonGroup>", () => {
       wrapper
         .find("RadioButton")
         .find("input")
-        .simulate("change");
+        .invoke("onChange")({ target: { value: "a" } });
```

Comment:
could do something similar to `assert.deepEqual(onChange.event, {target:{value:'a'})`